opengrok-indexer/src/main/java/org/opengrok/indexer/web/messages/MessagesUtils.java

```diff
@@ -133,7 +133,7 @@ private static String taggedMessagesToJson(Set<TaggedMessagesContainer> messages
             return mapper.writeValueAsString(messages);
         } catch (JsonProcessingException e) {
             LOGGER.log(Level.WARNING, "failed to encode {} to JSON", messages);
-            return null;
+            return empty;
```

Comment:
I think constant should be always named with capital letters. Our checkstyle does not enforce it?

Reply:
seems not